src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Interfaces/IUserInviteRepository.cs

```diff
@@ -0,0 +1,15 @@
+﻿#nullable enable
+using System;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Areas.Identity.Data.Models;
+using GovUk.Education.ExploreEducationStatistics.Admin.Models;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces
+{
+    public interface IUserInviteRepository
+    {
+        public Task<UserInvite> Create(string email, Role role, Guid createdById);
+
+        public Task<UserInvite> Create(string email, string roleId, Guid createdById);
```

Comment:
Really minor thing, but we don't need to include `public` in interface definitions

Reply:
Fixed. I think that's me copying and pasting from the class.